sections/futures/Trade/NextPrice.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// import Button from 'components/Button';
+// import Input from 'components/Input/Input';
+
+const NextPrice: React.FC = () => {
+	// const [volatility, setVolatility] = React.useState('');
+	// const inputRef = React.useRef<HTMLInputElement>(null);
+
+	// React.useEffect(() => {
+	// 	if (volatility.includes('%')) {
+	// 		inputRef?.current?.setSelectionRange(volatility.length - 1, volatility.length - 1);
+	// 	}
+	// }, [volatility]);
+
+	return (
+		<NextPriceContainer>
+			<p className="next-price-description">
+				Next-Price orders are subject to volatility and execute at the very next on-chain price.{' '}
+				<a href="https://example.com" rel="noreferrer" target="_blank">
```

Comment:
Instead of hardcoding the URL, can you add it to `constants/links.ts` and point to it from here so we can update it on-the-fly once done?

Reply:
Done @platschi